charts/opentelemetry-collector/values.yaml

```diff
@@ -312,6 +312,12 @@ serviceMonitor:
   extraLabels: {}
   #  release: kube-prometheus-stack
 
+# PodDisruptionBudget is used only if standaloneCollector enabled
+PodDisruptionBudget:
```

Comment:
Should not be capitalized `podDisruptionBudget` according to helm style guides

Reply:
fixed